lya/data.py

```diff
@@ -40,10 +41,30 @@ def plot_pk(params=None, filename=lya_data_path):
 
 
 def get_data_transfer_function():
+    
+   
     """Returns the data "transfer function" 
 
     Returns
     -------
     data : Data object
         Data object representing the ly-a data divided by LCDM p(k)
     """
+    
+    # 
+    pkncdm_data = load_lya_data()
+    
+    lcdm_pk = get_lcdm_pk(pkncdm_data.x)
+    
+    tk = np.sqrt(pkncdm_data.y / lcdm_pk)
+    
+    tk_unc = 0.5 * pkncdm_data.y_unc / lcdm_pk
```

Comment:
This doesn't look quite right...for instance, it doesn't have the right dimensions---as it is now, `tk_unc` is a dimensionless quantity, since both pk and pk_unc have the same dimensions.  Check out the "example formulae" section of [this wikipedia article](https://en.wikipedia.org/wiki/Propagation_of_uncertainty#Non-linear_combinations) on propogation of uncertainty; I think this corresponds to the f = aA^b case.

Reply:
Yes, of course Tk is dimensionless, sorry! [face palm.] But shouldn't it be `pkncdm_data.y` (instead of `x`)?

Reply:
yep! My bad...